projects/media-viewer/src/lib/toolbar/main-toolbar/left-pane/left-pane.component.html

```diff
@@ -25,11 +25,14 @@
          tabindex="0" data-l10n-id="page" (change)="onPageNumberInputChange(pageNumberInput.value)" #pageNumberInput>
     <span id="numPages" class="toolbarLabel"></span>
   </ng-container>
-  <div class="toolbarButtonSpacer"></div>
```

Comment:
might have been a reason why this is here, i dont remember but i know its in other parts of the toolbar html (mostly likely styles).

Reply:
I'll re-add.